core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4230,10 +4230,6 @@ int RootClingMain(int argc,
 #endif
    clingArgs.push_back("-Xclang");
    clingArgs.push_back("-fmodules-embed-all-files");
-   clingArgs.push_back("-Xclang");
```

Comment:
Can you check what the error output is when loading broken code through the dictionary? This used to say "included from <dictionary file>" and this commit might make this less informative?

Reply:
Can you tell me how to do that?

Reply:
Dunno, maybe generating the dict for a header and then removing it? Or generating the dictionary and then `#define`-ing something that breaks the header?